discovery/syncer.go

```diff
@@ -18,14 +18,21 @@ import (
 type SyncerType uint8
 
 const (
-	// ActiveSync denotes that a gossip syncer should exercise its default
-	// behavior. This includes reconciling the set of missing graph updates
-	// with the remote peer _and_ receiving new updates from them.
+	// ActiveSync denotes that a gossip syncer:
+	//
+	//   1. Should not attempt to synchronize with the remote peer for
+	//   missing channels.
+	//   2. Should respond to queries from the remote peer.
+	//   3. Should receive new updates from the remote peer.
+	//
+	// They are started in a chansSynced state in order to accomplish their
```

Comment:
can we remove `chansSynced` now?

Reply:
What do you mean by remove? Just remove that part of the comment?

Reply:
Was just wondering whether the `chansSynced` variable is possible to remove, if they always start in this state now.

Reply:
Don't think so as this is the only state in which we can process sync transitions.